updatecli/weekly.d/jenkinscontroller-cert-ci-maven.yaml

```
@@ -0,0 +1,175 @@
+---
+name: Bump Maven version for cert.ci.jenkins.io
+
+scms:
+  default:
+    kind: github
+    spec:
+      user: "{{ .github.user }}"
+      email: "{{ .github.email }}"
+      owner: "{{ .github.owner }}"
+      repository: "{{ .github.repository }}"
+      token: "{{ requiredEnv .github.token }}"
+      username: "{{ .github.username }}"
+      branch: "{{ .github.branch }}"
+
+  gitMaven:
+    kind: "git"
+    spec:
+      url: "https://github.com/apache/maven.git"
+      branch: "master"
+
+sources:
+  mavenVersion3-5:
+    kind: gittag
+    name: Get the latest Maven 3.5.x version
+    scmid: gitMaven
+    spec:
+      versionfilter:
+        kind: regex
+        # Only full releases (semver with "maven-" prefix")
+        pattern: "^maven-3.5.(\\d*)$"
```

```suggestion
        pattern: "^maven-3\\.5\\.(\\d*)$"
```

(nit) let's be strict and specify literal dot by escaping them